CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Breaking Changes
 
+* `Removal of Python 3.9 support`: Due to upcoming changes to training requiring the usage of [GPTDolomite](https://github.com/instructlab/GPTDolomite), Python 3.9 support has been removed starting with this version. If you are using Python 3.9, you will need to upgrade to Python 3.10 or later to use this and subsequent versions of the CLI
```

Comment:
```suggestion
* `Removal of Python 3.9 support`: Due to upcoming changes to training requiring the usage of [GPTDolomite](https://github.com/instructlab/GPTDolomite), Python 3.9 support has been removed starting with this version. If you are using Python 3.9, you will need to upgrade to Python 3.10 or 3.11 to use this and subsequent versions of the CLI
```

Reply:
Right now README.md says this
```text
- Python 3.10+ (<3.12 for PyTorch JIT)
```

Reply:
OK -- not with the training library, it think -- but some configs will still work.

that should be clarified once the training library is actually integrated, though